murdock/database/__init__.py

```diff
@@ -0,0 +1,43 @@
+import abc
+from typing import List, Optional
+
+from murdock.job import MurdockJob
+from murdock.models import JobModel, JobQueryModel
+
+
+class Database(abc.ABC):
+    @abc.abstractmethod
+    def __init__(self):
+        pass
+
+    @abc.abstractmethod
+    async def init(self):
+        pass
```

Comment:
I prefer the `...` notation :)
Also you can skip the coverage using a pragma comment (same comment in other abstract methods below).

```suggestion
        ...  # pragma: no cover
```

Reply:
I annotated all functions with docstring, that also removes the need for `pass` and `...`. Probably doesn't satisfy the coverage, but lets see.